schemas/iso19110/src/main/plugin/iso19110/present/metadata-view.xsl

```diff
@@ -189,7 +191,12 @@
               </th>
               <td>
                 <xsl:value-of select="gfc:memberName/gco:LocalName"/>
-
+                <xsl:choose>
```

Comment:
I don't think those file in present/*.xsl are used anywhere in GeoNetwork 3?

Reply:
thanx, i was wondering about it, should we remove it?

Reply:
Those old stuffs have been removed in 4.0.x